MakeItGif

```diff
@@ -1,26 +1,36 @@
 #!/bin/bash
 if [ $# -lt 3 ]
-	then
-		echo "MakeItGif v1.0.0 , Makes gif out of videos using ffmpeg."
-		echo "Copyright (c) 2017 antony jr."
-		echo " "
-		echo "Usage: MakeItGif [VIDEO FILE] [SCALE] [OUTPUT]"
-		exit -1
+    then
+        echo "MakeItGif v1.0.0 , Makes gif out of videos using ffmpeg."
+        echo "Copyright (c) 2017 antony jr."
+        echo " "
+        echo "Usage: MakeItGif [VIDEO FILE] [SCALE] [OUTPUT]"
+        exit -1
 fi
 
+#Check If ffmpeg Is Installed
+if ! [ -x "$(command -v ffmpeg)" ]; then
+printf "ffmpeg not installed , Installing ffmpeg\n"
+
+apt-get update > /dev/null 2>&1
```

Comment:
we do not need to install ffmpeg , just exit as a error because its not sure that every linux user uses **apt-get** package manager.